pandamium_datapack/data/pandamium/functions/misc/spawnpoint/check_existence.mcfunction

```diff
@@ -0,0 +1,5 @@
+tag @s remove spawnpoint_advancement
+
+data modify storage pandamium: player set from entity @s

```

Comment:
Why move the nbt to storage then set the scores? Why not just check the player's nbt? I assume so that there aren't entities involved, it just seems needlessly complicated

Reply:
Also, you should call the storage `pandamium:spawnpoint` or something rather than `pandamium:`

Reply:
Every entity nbt data access serializes the whole entity's nbt data, so it's quite slow, especially if the player has e.g. a lot of recipes unlocked/items with a lot of data in their inventory/ender chest.
Accessing storage can be about as fast as accessing scores.